website/docs/cloud-docs/workspaces/settings/deletion.mdx

```diff
@@ -47,9 +54,22 @@ To cancel a scheduled auto destroy run in Terraform Cloud:
 1. Under **Automatically destroy**, click **Edit** next to your scheduled run's details.
 1. Click **Remove**.
 
-You can schedule [notifications](/terraform/cloud-docs/workspaces/settings/notifications) to alert you 12 and 24 hours before an auto destroy run, and to report auto destroy run results.
+#### Destroy if Inactive
+
+You can configure Terraform Cloud to automatically destroy a workspace's infrastructure after a period of inactivity. Any runs that update Terraform state will push back the scheduled auto destroy time by the configured activity duration designated by the user.
```

Comment:
Naive question, but can _other_ workspace affect "inactivity" for a workspace?

Reply:
I'd love to get super clear on what "inactivity" for a workspace looks like with something like the below suggestion.

```suggestion
You can configure Terraform Cloud to automatically destroy a workspace's infrastructure after a period of inactivity. A workspace is _inactive_ if the workspace's state has not changed within your designated time period.

If you configure a workspace to auto-destroy its infrastructure when inactive, any run that updates Terraform state further delays the scheduled auto-destroy time by the length of your designated time period.
```

Reply:
>Naive question, but can other workspace affect "inactivity" for a workspace?

Not really, unless there are something like run triggers but those will trigger new runs for the workspace so that _should_ technically be captured here. But open to suggestions if you feel like it should be more explicit.

Reply:
That makes sense to me! Thank you for explaining the background here, I agree that what you've written covers things!